doc/api/stream.md

```diff
@@ -1886,8 +1886,9 @@ methods only.
 
 The `writable._writev()` method may be implemented in addition or alternatively
 to `writable._write()` in stream implementations that are capable of processing
-multiple chunks of data at once. If implemented, the method will be called with
-all chunks of data currently buffered in the write queue.
+multiple chunks of data at once. If implemented and if there are buffered data
+from previous writes, `_writev` (that is capable of handling chunks of data)
```

Comment:
I think the previous sentence implies that `_writev()` can handle chunks of data, and there’s no need to duplicate that:

```suggestion
from previous writes, `_writev`
```

Reply:
@addaleax, Thanks. Fixed it.